cime_config/buildnml

```diff
@@ -1578,7 +1582,11 @@ cat >>! $RUNDIR/ocn_in$inststr << EOF
 !   EXPOCA         - Ca export production (epcalc100) [mol Ca m-2 s-1]
 !   EXPOSI         - Si export production (epsi100) [mol Si m-2 s-1]
 !   PCO2           - Surface PCO2 (spco2) [uatm]
-!   KWCO2          - kwco2 x solubility
+!   PCO2M          - Surface PCO2 under moist air assumption [uatm]
+!   KWCO2          - kwco2 
+!   KWCO2KHM       - kwco2 x solubility (moist air)
+!   CO2KH          - khd CO2 solubility under dry air assumption 
```

Comment:
Would be nice to stick to the convention used for the other namamelist entries, e.g.

CO2KH     - CO2 solubility under dry air assumption (khd) [mol/kg/atm]

Reply:
I hope the last commit satisfies the conventions. I left the other comment untouched (`kwco2d`) for the mentioned reasons. I hope that's ok.